# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_1351443-missing-NewChannel2.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_1351443-missing-NewChannel2.js
--- a/netwerk/test/unit/test_1351443-missing-NewChannel2.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_1351443-missing-NewChannel2.js	Tue Feb 12 16:08:25 2019 +0000
@@ -78,21 +78,19 @@
     throw Components.Exception("Setting content length", NS_ERROR_NOT_IMPLEMENTED);
   },
   open: function() {
+    // throws an error if security checks fail
+    contentSecManager.performSecurityCheck(this, null);
+
     var file = do_get_file("test_bug894586.js", false);
     Assert.ok(file.exists());
     var url = Services.io.newFileURI(file);
-    return NetUtil.newChannel({uri: url, loadUsingSystemPrincipal: true}).open2();
-  },
-  open2: function() {
-    // throws an error if security checks fail
-    contentSecManager.performSecurityCheck(this, null);
-    return this.open();
+    return NetUtil.newChannel({uri: url, loadUsingSystemPrincipal: true}).open();
   },
   asyncOpen: function(aListener, aContext) {
     throw Components.Exception("Not implemented",
                                Cr.NS_ERROR_NOT_IMPLEMENTED);
   },
-  asyncOpen2: function(aListener, aContext) {
+  asyncOpen: function(aListener, aContext) {
     throw Components.Exception("Not implemented",
                                Cr.NS_ERROR_NOT_IMPLEMENTED);
   },